packages/code-du-travail-frontend/pages/recherche.js

```diff
@@ -42,10 +45,37 @@ class SearchPage extends React.Component {
           <title>{q} - Code du travail numérique</title>
         </Head>
         <Search />
-        <SearchResults query={q} data={results} source={source} />
+        <div className="section">
+          <div className="container">
```

Comment:
We have a `Container` component in the `@cdt/ui` no ? 
\ping @UnbearableBear

Reply:
thanks! I could use it !